src/utils/hooks/useKubevirtUserSettings/useKubevirtUserSettings.ts

```diff
@@ -25,20 +26,20 @@ const useKubevirtUserSettings: UseKubevirtUserSettings = (key) => {
   const [error, setError] = useState<Error>();
   const [userName, setUserName] = useState<string>();
   const [userSettings, setUserSettings] = useState<UserSettingsState>();
-  const [loading, setLoading] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [loadingUserName, setLoadingUserName] = useState<boolean>(true);
   const userConfigMapName = useMemo(() => `${userName}-kubevirt-settings`, [userName]);
 
   useEffect(() => {
     const fetchUserName = async () => {
       setError(null);
-      setLoading(true);
       try {
         const user = await k8sGet({ model: UserModel, path: '~' });
         setUserName(user?.metadata?.name?.replace(':', '-'));
       } catch (e) {
         setError(e);
```

Comment:
do we also want to set loading to false when we get an error?

Reply:
yes, loading isn't relevant to error, if u get an error loading is done, so its ok